problems/oldestPerson.js

```diff
@@ -11,8 +11,10 @@
  * ex: oldestPerson({ 'Marcus': 29, 'Julia': 29, 'Kevin': 34 })
  * returns: 'Kevin'
  */
-function oldestPerson() {
+function oldestPerson(obj) {
+    const oldestAge = 0
```

Comment:
This variable should be a let b/c we'll have to reassign it as we iterate through the object and find older people.